src/components/SecondaryPanes/Breakpoints.js

```diff
@@ -75,21 +75,79 @@ class Breakpoints extends PureComponent {
   }
 
   showContextMenu(e, breakpoint) {
-    const { removeBreakpoint, removeAllBreakpoints } = this.props;
+    const {
+      removeBreakpoint,
+      removeBreakpoints,
+      removeAllBreakpoints,
+      toggleBreakpoints,
+      toggleAllBreakpoints,
+      toggleDisabledBreakpoint,
+      breakpoints
+    } = this.props;
 
     e.preventDefault();
 
     const removeBreakpointLabel = L10N.getStr("breakpointMenuItem.deleteSelf");
     const removeAllBreakpointsLabel = L10N.getStr(
       "breakpointMenuItem.deleteAll"
     );
+    const removeOtherBreapointsLabel = L10N.getStr(
+      "breakpointMenuItem.deleteOthers"
+    );
+    const enableBreakpointLabel = L10N.getStr("breakpointMenuItem.enableSelf");
+    const enableAllBreakpointsLabel = L10N.getStr(
+      "breakpointMenuItem.enableAll"
+    );
+    const enableOtherBreakpointsLabel = L10N.getStr(
+      "breakpointMenuItem.enableOthers"
+    );
+    const disableBreakpointLabel = L10N.getStr(
+      "breakpointMenuItem.disableSelf"
+    );
+    const disableAllBreakpointsLabel = L10N.getStr(
+      "breakpointMenuItem.disableAll"
+    );
+    const disableOtherBreakpointsLabel = L10N.getStr(
+      "breakpointMenuItem.disableOthers"
+    );
 
     const removeBreakpointKey = L10N.getStr(
       "breakpointMenuItem.deleteSelf.accesskey"
     );
     const removeAllBreakpointsKey = L10N.getStr(
       "breakpointMenuItem.deleteAll.accesskey"
     );
+    const removeOtherBreapointsKey = L10N.getStr(
+      "breakpointMenuItem.deleteOthers.accesskey"
+    );
+    const enableBreakpointKey = L10N.getStr(
+      "breakpointMenuItem.enableSelf.accesskey"
+    );
+    const enableAllBreakpointsKey = L10N.getStr(
+      "breakpointMenuItem.enableAll.accesskey"
+    );
+    const enableOtherBreakpointsKey = L10N.getStr(
+      "breakpointMenuItem.enableOthers.accesskey"
+    );
+    const disableBreakpointKey = L10N.getStr(
+      "breakpointMenuItem.disableSelf.accesskey"
+    );
+    const disableAllBreakpointsKey = L10N.getStr(
+      "breakpointMenuItem.disableAll.accesskey"
+    );
+    const disableOtherBreakpointsKey = L10N.getStr(
+      "breakpointMenuItem.disableOthers.accesskey"
+    );
```

Comment:
this is the pattern we have elsewhere, but i'm wondering if we can improve upon it:

```js
const getKey = (key) => L10N.getStr(`breakpointMenuItem.${key}.accesskey `)
const getLabel = (label) => L10N.getStr(`breakpointMenuItem.${label} `)

const foo = {
  label: getLabel("foo")
  accesskey: getKey("foo")
}
```